src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/OverseasEntitySubmissionDto.java

```diff
@@ -250,6 +255,18 @@ public void setIsSecureRegister(Boolean isSecureRegister) {
         this.isSecureRegister = isSecureRegister;
     }
 
+    public WhoIsRegisteringType getWhoIsRegistering() {
+        return whoIsRegistering;
+    }
+
+    public void setWhoIsRegistering(String whoIsRegistering) {
+        if (whoIsRegistering != null) {
```

Comment:
Do we need the null check? Doesn't it have to be one or the other.

Reply:
For the moment this is null as long as redis has not been removed and the feature flag is not active on the web side
And it can be null during the update journey
Then the validator throws an exception in the case of null

Reply:
i think we need it as current records stored on mongo will have a null value, and we need to be able to retrieve existing records on mongo via the api.
Also, during the journey, pages before this one will be sent to the api before this value has been entered.